bitnami/mongodb/templates/standalone/dep-sts.yaml

```
@@ -158,6 +158,11 @@ spec:
               DNS.3 = $my_hostname.$svc.$MY_POD_NAMESPACE.svc.cluster.local
               DNS.4 = localhost
               DNS.5 = 127.0.0.1
+              {{- if .Values.tls.extraDnsNames }}
+              {{- range $index, $dnsName := .Values.extraDnsNames }}
+              - {{ $dnsName }}
+              {{- end }}
+              {{- end }}
```

Testing the solution I have realized that you missed the 'tls'  in 
`{{- range $index, $dnsName := .Values.extraDnsNames }}`
following `DNS.2 = $my_hostname` maybe this makes more sense, what do you think?
```suggestion
             {{- if .Values.tls.extraDnsNames }}
              {{- range $key, $dnsName := .Values.tls.extraDnsNames }}
              {{ $key }} = {{ $dnsName }}
              {{- end }}
              {{- end }}
```

The values would be something like this:
```yaml
  extraDnsNames: 
    "DNS.6": "$my_host"
    "DNS.7": "$test"
```

---

ho miss my mistake, didn't updated my code but you're right !

For 
> DNS.2 = $my_hostname maybe this makes more sense, what do you think?

I would say it's up to your preference. I prefer just sending a list, smaller to put in the values.yaml but I understand that for the debug it's may be more easy

---

If you check the actual appears something like this:
https://github.com/bitnami/charts/blob/2afda168725556251dc543ed81cee8b66b331d44/bitnami/mongodb/templates/standalone/dep-sts.yaml#L156-L160
So If you want to add more lines in `[alt_names]` I think that will be similar to the others.
Anyway, it would be nice to add an example in the values.yaml,

---

understood, didn't put the DNSx:  in my test and it was working but for uniformity I approved the commit